packages/paste-libraries/dropdown/stories/multiselect.stories.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import type {Story, Meta} from '@storybook/react';
+import {useCombobox, useMultipleSelection} from '../src';
+
+type Book = {
+  author: string;
+  title: string;
+};
+
+const books: Book[] = [
+  {author: 'Harper Lee', title: 'To Kill a Mockingbird'},
+  {author: 'Lev Tolstoy', title: 'War and Peace'},
+  {author: 'Fyodor Dostoyevsy', title: 'The Idiot'},
+  {author: 'Oscar Wilde', title: 'A Picture of Dorian Gray'},
+  {author: 'George Orwell', title: '1984'},
+  {author: 'Jane Austen', title: 'Pride and Prejudice'},
+  {author: 'Marcus Aurelius', title: 'Meditations'},
+  {author: 'Fyodor Dostoevsky', title: 'The Brothers Karamazov'},
+  {author: 'Lev Tolstoy', title: 'Anna Karenina'},
+  {author: 'Fyodor Dostoevsky', title: 'Crime and Punishment'},
+];
+const initialSelectedItems = [books[0], books[1]];
+
+function getFilteredBooks(selectedItems: Book[], inputValue: string): Book[] {
+  const lowerCasedInputValue = inputValue.toLowerCase();
+
+  return books.filter(function filterBook(book) {
+    return (
+      !selectedItems.includes(book) &&
+      (book.title.toLowerCase().includes(lowerCasedInputValue) ||
+        book.author.toLowerCase().includes(lowerCasedInputValue))
+    );
+  });
+}
+
+export const MultiselectComboBox: Story = () => {
```

Comment:
praise: really glad we've got some library stories now, for combobox but especially multiselect!